packages/disclosure/styles.css

```diff
@@ -0,0 +1,51 @@
+:root {
+  --reach-disclosure: 1;
+}
+
+[data-reach-disclosure] {
+  border: 1px solid #c5c5c5;
+}
+
+[data-reach-disclosure-panel] {
+  padding: 0.5em;
+  margin: 0;
+  border: 1px solid #aaa;
+  border-top: 0;
+  border-bottom-left-radius: 0.25rem;
+  border-bottom-right-radius: 0.25rem;
+}
+
+[data-reach-disclosure-trigger] {
```

Comment:
I think there are too many rules in these styles, just leave it as a button and give them an example in the docs of the styles to make buttons look like divs.

Reply:
Actually, I think we don't need any styles at all for this one.